daprdocs/content/en/reference/components-reference/supported-state-stores/_index.md

```diff
@@ -26,38 +26,41 @@ The following stores are supported, at various levels, by the Dapr state managem
+| [AWS DynamoDB]({{< ref setup-dynamodb.md>}})       | ✅   | ❌             | ❌ | ❌ | ❌ | ❌ | Alpha | v1 | 1.0 |
 
 ### Google Cloud Platform (GCP)
-| Name                                                           | CRUD | Transactional | ETag | [TTL]({{< ref state-store-ttl.md >}}) | [Actors]({{< ref howto-actors.md >}}) | Status | Component version | Since |
-|------------------------------------------------------------------|------|---------------------|------|-----|--------|-----|-----|-------|
-| [GCP Firestore]({{< ref setup-firestore.md >}})       | ✅   | ❌                  | ❌     | ❌ | ❌     | Alpha  | v1 | 1.0 |
+| Name                                               | CRUD | Transactional | ETag | [TTL]({{< ref state-store-ttl.md >}}) | [Actors]({{< ref howto-actors.md >}}) | [State Query API]({{< ref howto-state-query-api.md >}}) | Status | Component version | Since |
+|----------------------------------------------------|------|---------------|-----|-----|-----|-----|-------|----|-----|
+| [GCP Firestore]({{< ref setup-firestore.md >}})    | ✅   | ❌             | ❌  | ❌   | ❌  | ❌  | Alpha | v1 | 1.0 |
 
 ### Microsoft Azure
 
-| Name                                                           | CRUD | Transactional | ETag | [TTL]({{< ref state-store-ttl.md >}}) | [Actors]({{< ref howto-actors.md >}}) | Status | Component version | Since |
-|------------------------------------------------------------------|------|---------------------|------|-----|--------|-----|-----|-------|
-| [Azure Blob Storage]({{< ref setup-azure-blobstorage.md >}})     | ✅   | ❌                  | ✅    | ❌ | ❌    | Stable | v1 | 1.0 |
-| [Azure CosmosDB]({{< ref setup-azure-cosmosdb.md >}})            | ✅   | ✅                  | ✅    | ✅ | ✅    | Stable | v1 | 1.0 |
-| [Azure SQL Server]({{< ref setup-sqlserver.md >}})               | ✅   | ✅                  | ✅    | ❌ | ✅    | Alpha  | v1 | 1.0 |
-| [Azure Table Storage]({{< ref setup-azure-tablestorage.md >}})   | ✅   | ❌                  | ✅    | ❌ | ❌    | Alpha  | v1 | 1.0 |
+| Name                                                             |CRUD|Transactional|ETag| [TTL]({{< ref state-store-ttl.md >}}) | [Actors]({{< ref howto-actors.md >}}) | [State Query API]({{< ref howto-state-query-api.md >}}) | Status | Component version | Since |
+|------------------------------------------------------------------|----|-------------|----|----|----|----|-------|----|-----|
+| [Azure Blob Storage]({{< ref setup-azure-blobstorage.md >}})     | ✅ | ❌           | ✅ | ❌ | ❌ | ❌ | Stable | v1 | 1.0 |
+| [Azure CosmosDB]({{< ref setup-azure-cosmosdb.md >}})            | ✅ | ✅           | ✅ | ✅ | ✅ | 🔶 | Stable | v1 | 1.0 |
+| [Azure SQL Server]({{< ref setup-sqlserver.md >}})               | ✅ | ✅           | ✅ | ❌ | ✅ | ❌ | Alpha  | v1 | 1.0 |
+| [Azure Table Storage]({{< ref setup-azure-tablestorage.md >}})   | ✅ | ❌           | ✅ | ❌ | ❌ | ❌ | Alpha  | v1 | 1.0 |
+
+###
+🔶 feature in **alpha** stage.
```

Comment:
Oh! Now I discover what this means!! I would still use a ✅ since that is the eventual state and you have said in many other places that this is in alpha stage, so people are not going to get confused. I would not introduce a new symbol into the table.